packages/nhsuk-frontend/src/nhsuk/components/header/_index.scss

```diff
@@ -60,14 +61,22 @@ $_header-item-padding: 12px;
 
 .nhsuk-header__service-logo {
   flex-grow: 0;
+  margin-right: 0;
 }
 
 .nhsuk-header__service-name {
-  flex-grow: 999;
+  flex-grow: 0;
+  margin-right: auto;
   text-wrap: balance;
   @include nhsuk-font(19, $line-height: 1.1);
 }
 
+.nhsuk-header__service-logo[href] {
+  @include nhsuk-link-image;
+
+  display: inline-flex;
```

Comment:
I thought this might trigger a `mixed-dclr` warning, but I need to override the display property used by `nhsuk-link-image`, and got no listing warnings, so maybe this is fine? @colinrotherham